client/components/filters/advanced/product-search.js

```diff
@@ -0,0 +1,57 @@
+/** @format */
```

Comment:
Can this file be moved to `components/search`? It doesn't seem to be filter-specific, and it might be useful for other pages/third-party devs to have access to a `<ProductSearch />`